ogame.go

```diff
@@ -185,7 +185,7 @@ type DefenderObj interface {
 // Ship interface implemented by all ships units
```

Comment:
Look at line 2410 in this file.
SendFleet needs to calculate the cargo, and is currently using the (base) "CargoCapacity" not the "GetCargoCapacity" function.
You need to rename "CargoCapacity" to "BaseCargoCapacity" for example and see everything that it will break.
At that point, you'll be able to figure out who is using the class property directly and replace them with the "GetCargoCapacity" function